backend/src/main/java/CustomerController.java

```diff
@@ -0,0 +1,88 @@
+import model.*;
+import storage.DBInterface;
+import storage.PostgresSQLConnector;
+import storage.repository.GenericRepository;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use classes from the Java API instead of Sun classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1191)